apps/examples/src/misc/develop.tsx

```diff
@@ -108,12 +113,18 @@ export default function Develop() {
 				onMount={(editor) => {
 					;(window as any).app = editor
 					;(window as any).editor = editor
```

Comment:
I think it's be a lot simpler to just add a getter on window?

```ts
Object.defineProperty(window, '$s', {
  get: function () {
    return editor.getOnlySelectedShape()
  },
  configurable: true,
  enumerable: true,
});
```

Reply:
that works! 👍